### PR TITLE
Add function log_indices that returns number of currently active indices

### PR DESCRIPTION
**Pull request by WurmPeter**
Solving #180 

One of my very first pull requests. Please feel free to tell me anything I could do better.

### CODE REVIEW

File: R/logger.R [hunk 1]
```diff
@@ -275,6 +275,20 @@ log_namespaces <- function() {
 }
 
 
+#' Returns number of currently active indices
+#' 
+#' @param namespace string referring to the `logger` environment /
+#'   config to be used to override the target of the message record to
+#'   be used instead of the default namespace, which is defined by the
+#'   R package name from which the logger was called, and falls back
+#'   to a common, global namespace.
```
Comment (daroczig) [inline R/logger.R:284]: No need to copy the whole `namespace` param description, as you can reference it from another fn to be reused, e.g.:

```suggestion
#' @inheritParams get_logger_definitions
```

File: R/logger.R [hunk 2]
```diff
@@ -275,6 +275,20 @@ log_namespaces <- function() {
...
+#' @return number of indices
+#' @export
+log_indices <- function(namespace = "global") {
+  length(get(logger:::fallback_namespace(namespace), envir = logger:::namespaces))
```
Comment (daroczig) [inline R/logger.R:288]: no need for the triple colons as calling from inside the pkg

```suggestion
  length(get(fallback_namespace(namespace), envir = namespaces)) 
```

**Comment by daroczig** [comment]
Thanks a ton :raised_hands: 

I've left a few minor comments -- would you mind reviewing those, and also run roxygen to generate the `Rd` files? No worries if that's too much, I can also do that, just let me know.

**Comment by WurmPeter** [comment]
I have made a new commit. Please review again :)

**Comment by daroczig** [comment]
Thank you! Could you please double check on https://github.com/daroczig/logger/pull/194#discussion_r1714365782? I don't think any colons are needed at all.

**Comment by daroczig** [comment]
no worries 🙌 
yes, pls. would be also nice to update the pkgdown.yaml file as per the related error msg (mention among the other related fans)

**Comment by WurmPeter** [comment]
Great, thanks for your patience.

**Pull request merged**